package.json

```diff
@@ -62,48 +65,57 @@
     "react-dom": ">=16.0.0"
   },
   "dependencies": {
-    "prop-types": "^15.6.0"
+    "aphrodite-jss": "^2.1.0",
+    "classnames": "^2.2.6",
```

Comment:
classnames package was already removed for maintain the package be not dependable for other packages.

Reply:
Classnames removed.